controllers/restaurants.controller.js

```diff
@@ -4,7 +4,7 @@ const db = require('../db/index');
 exports.showRestaurants = async (req, res, next) => {
   try {
     const restaurants = await queryDbFromReqQuery(
-      'SELECT DISTINCT rname FROM OwnedRestaurants NATURAL JOIN HasTimeslots',
+      'SELECT rname, cuisine FROM OwnedRestaurants ORDER BY rname',
```

Comment:
Actually this change will break the search; but I'll fix it in the next PR so shouldn't be a problem.